test/module.test.js

```diff
@@ -128,6 +128,32 @@ describe('basic', () => {
   })
 })
 
+describe('with redirect route', () => {
+  let nuxt
+
+  beforeAll(async () => {
+    const override = {
+      router: {
+        extendRoutes (routes) {
```

Comment:
Sorry to be nitpicking more but I've just realized that we could merge this test into 'basic' one. Every new fixture build adds a lot of time to testing so if we can avoid that, it would be preferred.